WikipediaUnitTests/Code/MWKImageInfo+MWKImageComparisonTests.m

```diff
@@ -21,16 +13,16 @@ @interface MWKImageInfo_MWKImageComparisonTests : XCTestCase
 @implementation MWKImageInfo_MWKImageComparisonTests
 
 - (void)testAssociation {
-    MWKImage* image    = [MWKImage imageAssociatedWithSourceURL:@"some_file_name.jpg/400px-some_file_name.jpg"];
-    MWKImageInfo* info = [MWKImageInfo infoAssociatedWithSourceURL:@"some_file_name.jpg/800px-some_file_name.jpg"];
+    MWKImage* image    = [MWKImage imageAssociatedWithSourceURL:@"/thumb/some_file_name.jpg/400px-some_file_name.jpg"];
```

Comment:
@joewalsh why do we need to fix the test? 

Worried that the tests are changing for images that aren't PDFs. Did we need to update any assumptions about the paths in the codebase?